db/migrate/20160420113806_create_asset_directories.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAssetDirectories < ActiveRecord::Migration
+  def change
+    create_table :asset_directories do |t|
```

Comment:
Ideally you should be going back and making these three migrations into one migration, because all you did was make a table. But instead of having the one migration, you have a migration, then adding columns, then changing the table name, in one merge. You could be merging in just one migration to do essentially the same as the three.
But for the sake of progress, you can move on with it like this.